source/_integrations/influxdb.markdown

```diff
@@ -37,41 +40,63 @@ You will still need to create a database named `home_assistant` via InfluxDB's c
 {% configuration %}
 host:
   type: string
-  description: IP address of your database host, e.g., 192.168.1.10
+  description: **1.xx only** IP address of your database host, e.g., 192.168.1.10
```

Comment:
If you start the description with a `*`, you need to quote from begin to end with `" "`

```suggestion
  description: "**1.xx only** IP address of your database host, e.g., 192.168.1.10"
```

Reply:
Fixed. And thank you!  That error message was driving me crazy, I did not know what it was talking about with the line 4 column 16 bit.

Reply:
@klaasnicolaas this issue should be resolved now but I can't seem to mark it as such, still showing up as a requested change blocking merge. Can you resolve this one? Just trying to get everything up for a merge now that the implementation is nearing completion.